Domain.Sql/ReadModelDatabaseInitializer.cs

```diff
@@ -14,8 +14,41 @@ namespace Microsoft.Its.Domain.Sql
     /// <typeparam name="TDbContext">The type of the db context.</typeparam>
     public class ReadModelDatabaseInitializer<TDbContext> : DropCreateDatabaseIfModelChanges<TDbContext>
         where TDbContext : ReadModelDbContext, new()
-    {
-       
+    {
+        /// <summary>
+        /// See https://msdn.microsoft.com/en-us/library/dn268335.aspx for more info.
+        /// 
+        /// MAXSIZE = ( [ 100 MB | 500 MB ] | [ { 1 | 5 | 10 | 20 | 30 � 150�500 } GB  ] )
+        /// | EDITION = { 'web' | 'business' | 'basic' | 'standard' | 'premium' } 
+        /// | SERVICE_OBJECTIVE = { 'shared' | 'basic' | 'S0' | 'S1' | 'S2' | 'P1' | 'P2' | 'P3' } 
+        /// </summary>
+        /// <param name="context"> The DbContext </param>
+        /// <param name="dbSizeInGB"> Size of database in GB </param>
+        /// <param name="edition"> Edition of database </param>
+        /// <param name="serviceObjective"> Service objective of database </param>
+        public void InitializeDatabase(TDbContext context, int dbSizeInGB, string edition, string serviceObjective)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException("context");
+            }
+
+            bool flag;

```

Comment:
A name that explains the intent is generally preferable, but since you're only using it once, it's cleaner just to inline the `context.Database.Exists()` expression.